bingo/oracle/src/oracle/ringo_oracle.cpp

```diff
@@ -16,14 +16,13 @@
  * limitations under the License.
  ***************************************************************************/
 
-#include "oracle/ora_wrap.h"
-#include "oracle/ringo_oracle.h"
 #include <memory>
-#include "base_cpp/output.h"
+
 #include "oracle/bingo_oracle_context.h"
+#include "oracle/ora_wrap.h"
+#include "oracle/ringo_oracle.h"
 #include "oracle/ringo_shadow_table.h"
```

Comment:
Also here: one include file lost.

Reply:
after reordering some includes became redundant and so've been deleted.